### PR TITLE
more common core codes

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: warehouse/sql/V201707251501021717__more_code_cleanup.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- more code cleanup
+
+USE ${schemaName};
+
+-- Although we'd like to do this, we can't with existing data
+-- If a full data wipe is done (and scripts consolidated) most UNKNOWN claim targets should be eliminated
+-- DELETE FROM target WHERE description = 'UNKNOWN' AND natural_id NOT IN ('S-ID|P', 'G-SRT|O');
```
Comment (malaffoon) [inline warehouse/sql/V201707251501021717__more_code_cleanup.sql]: This is a little annoying. I think we're going to have to actually reload all assessments to get the corrected secondary claim targets. @agorina, perhaps we can discuss when we have some free time?

Comment (agorina) [inline warehouse/sql/V201707251501021717__more_code_cleanup.sql]: We can discuss it any time you want to.

File: warehouse/sql/V201707251501021717__more_code_cleanup.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+INSERT INTO common_core_standard (subject_id, natural_id, description) VALUES
```
Comment (chdietz) [inline warehouse/sql/V201707251501021717__more_code_cleanup.sql]: These are the code you discovered missing as mentioned I guess. Are any of these duplicates, in other words do they exist with another natural id?

**Comment by malaffoon** [comment]
@chdietz, regarding the delete ...

When i import the latest tabulator output for the last three years, there are only 2 of the UNKNOWN claim targets referenced. However, in the existing databases there are 526 references to the UNKNOWN claim targets. It has to do with how the tabulator was parsing the assessment packages, that has subsequently been fixed.

File: warehouse/sql/V201707251501021717__more_code_cleanup.sql (hunk 2 above)
Comment (malaffoon) [inline warehouse/sql/V201707251501021717__more_code_cleanup.sql]: Nope, they are all non-specific category-level codes that didn't exist. But, for example, G-CO.A was already in our table even though it is such a beast and really shouldn't be there. I guess AIR put it in.

**Pull request merged**